source/gui/__init__.py

```diff
@@ -81,7 +81,6 @@ def __init__(self):
 				self.Hide()
 
 	def _onDestroy(self, evt):
-		evt.Skip()  # Allow other handlers to process this event.

```

Comment:
Why should `evt.Skip()` be removed? There shouldn't be any other parent controls, but if there were?

Reply:
The install process was crashing, so I moved to safer behaviour of stopping and resuming the event propagation.

Reply:
I've added a comment that makes the reason for this more explicit